src/features/cartography/use-cases/list-cnfs/list-cnfs-positions.use-case.ts

```diff
@@ -0,0 +1,18 @@
+import type { Observable } from 'rxjs';
+import type { UseCase } from '../../../common/use-case';
+import type { FeatureCollection as GeoJsonFeatureCollection } from 'geojson';
+import type { CnfsRepository } from '../../core';
+import { map } from 'rxjs';
+import { cnsfCoreToPresentation } from '../../infrastructure/presentation';
+
+export class ListCnfsPositionUseCase
+  implements UseCase<[string, string], GeoJsonFeatureCollection>
+{
+  public constructor(
+    private readonly cnfsApi: CnfsRepository
+  ) {}
+
+  public execute$(): Observable<GeoJsonFeatureCollection> {
```

Comment:
```suggestion
  public execute$(): Observable<ListCnfsPositions> {
```

Dans les  models : `export type ListCnfsPositions = GeoJsonFeatureCollection`